src/components/v2/MarkdownRender.vue

```diff
@@ -0,0 +1,99 @@
+<script lang="ts" setup>
+import { computed } from 'vue';
+import markdownit from 'markdown-it';
+import { markdownItTable } from 'markdown-it-table';
+import LinkProvider from 'src/providers/components/LinkProvider';
+
+interface MarkdownRenderProps {
+    markdown: string;
+}
+
+const props = defineProps<MarkdownRenderProps>();
+
+const markdownToRender = computed(() => {
+    const md = markdownit("commonmark", {
+        linkify: true
+    }) as any;
+    md.use(markdownItTable);
+    md.enable(['linkify', 'strikethrough']);
+    return md.render(props.markdown);
+})
+
+function capturePropagationParentLink(target: HTMLElement, originalEvent: Event) {
+    if (target.tagName.toLowerCase() === "a") {
+        const href = target.getAttribute("href") || "";
+        if (!href.startsWith("#")) {
+            originalEvent.preventDefault();
+            LinkProvider.instance.openLink(target.getAttribute("href")!);
+        }
+    } else if (target.parentElement) {
+        capturePropagationParentLink(target.parentElement, originalEvent);
+    }
+}
+
+function captureClick(e: Event) {
+    if (e.target && e.target instanceof HTMLElement) {
+        capturePropagationParentLink(e.target, e);
+    }
+    return e;
+}
+</script>
+
+<template>
+    <p class="markdown-body" v-html="markdownToRender" @click="captureClick"></p>
+</template>
+
+<style lang="scss">
```

Comment:
Is the `scoped` attribute intentionally omitted here? `table th` looks like a rather generic selector. (I don't quite remember how the unscoped styles bleed around in Vue.)